project/npda/templates/npda/visit_form.html

```diff
@@ -61,4 +60,36 @@
       </form>
     </div>
   </div>
+  <script>
```

Comment:
This script was in `visit_form_tab` before so was run 3 times. I've moved it here to run once.

The links no longer have the `tab` parameter and we look up the right tab in the script from the anchor